provision/kubernetes/monitor.go

```diff
@@ -167,7 +167,11 @@ func (c *clusterController) startJobInformer() error {
 			if job.Labels["tsuru.io/is-tsuru"] != "true" {
 				return
 			}
-			createJobEventAndMetrics(job, evt)
+			wg := &sync.WaitGroup{}
```

Comment:
increment metrics on prometheus is so fast, you don't need to spawn a goroutine for this, you could remove waitGroup
@arthurcgc

Reply:
It's not about how fast it goes, it's idiomatic and clean. Your proposal seems more like a preference than a recommendation.

Reply:
Yeah, I prefer to spawn a goroutine when is favored, to avoid stimulus to other developers to use goroutine everywhere.

but it is a philosophical approach, your case is not harmful, cause is a small context of use of goroutine.